### ISSUE TITLE
Compatibility with Magento 2.4.4 and PHP 8.1

### ISSUE THREAD

**Issue by Davie82** [trustpilot/plugin-magento2#94]
Currently I am upgrading to Magento 2.4.4. When running `bin/magento setup:di:compile` I get the following error:
```
Deprecated:  Optional parameter $data declared before required parameter $pluginStatus is implicitly treated as a required parameter in /vendor/trustpilot/module-reviews/Block/Success.php on line 25
```

Magento version: 2.4.4
Module version: 2.6.566
PHP version: 8.1

**Comment by dsjacobsen** [comment]
Same issue registered here.

**Comment by vlad1992** [comment]
I am also seeing 1 exception(s):
Exception #0 (Exception): Deprecated Functionality: strlen(): Passing null to parameter #1 ($string) of type string is deprecated in .../vendor/trustpilot/module-reviews/Helper/Data.php on line 151

After manually switching the places of $data and $pluginStatus in a composer patch

**Comment by git-seb** [comment]
The right way to fix both issues is change /vendor/trustpilot/module-reviews/Helper/Data.php
line 151 to:
`        if (strlen($this->_request->getParam('store') ?? '')) {`
line 155 to:
`        if (strlen($this->_request->getParam('website') ?? '')) {`
and also change /vendor/trustpilot/module-reviews/Block/Success.php
line 32 to:
`        array $data,`

**Comment by vlad1992** [comment]
@git-seb are you sure adding more cyclomatic complexity for the sake of it is the right way to go as opposed to removing the call to strlen() that brings no additional benefit as far as I can see? 
`if (strlen($this->_request->getParam('store'))) {` and `if ($this->_request->getParam('store')) {` literally pass and fail for the same reasons. Am I missing something?

**Comment by arunas-stonis** [comment]
The issue was fixed in release [2.6.572](https://github.com/trustpilot/plugin-magento2/releases/tag/2.6.572). Thank you all!
